### ISSUE TITLE
cannot disable zoom on iOS

### ISSUE THREAD

**Issue by sophister** [iou90/react-native-autoheight-webview#112]
Hi, I want to **disable** zoom , but fails on `iOS` . 
this [line of source](https://github.com/iou90/react-native-autoheight-webview/blob/master/autoHeightWebView/utils.js#L34) maybe should be controlled by user:

```
// add viewport setting to meta for WKWebView
const makeScalePageToFit = `
var meta = document.createElement('meta'); 
meta.setAttribute('name', 'viewport'); 
meta.setAttribute('content', 'width=device-width'); document.getElementsByTagName('head')[0].appendChild(meta);
`;
```

**Comment by iou90** [comment]
'width=device-width' has nothing to do with zooming.
Can you zoom only with react-native-webview?

**Comment by sophister** [comment]
Hmm, I want to **disable** zoom on iOS. But since this library injects some script and creates a `<meta />` which enables use to zoom. I want to set `<meta />` like this
```html
<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
```

**Comment by iou90** [comment]
Should be fixed with @1.1.3.
